client/public/styles/project.scss

```diff
@@ -0,0 +1,29 @@
+.project {
+  width: 80vw;
+  margin-left: 50%;
+  transform: translateX(-50%);
+}
+
+.project-info {
+  text-align: center;
+  margin-top: 20px;
```

Comment:
We can fix this later, but we want to avoid using px values here so that our website can be as compatible as possible (pref % or em).